lib/node_modules/@stdlib/math/base/special/sinh/README.md

```diff
@@ -77,6 +77,95 @@ for ( i = 0; i < x.length; i++ ) {
 
 <!-- /.examples -->
 
+<!-- C interface documentation. -->
+
+* * *
+
+<section class="c">
+
+## C APIs
+
+<!-- Section to include introductory text. Make sure to keep an empty line after the intro `section` element and another before the `/section` close. -->
+
+<section class="intro">
+
+</section>
+
+<!-- /.intro -->
+
+<!-- C usage documentation. -->
+
+<section class="usage">
+
+### Usage
+
+```c
+#include "stdlib/math/base/special/sinh.h"
+```
+
+#### stdlib_base_sinh( x )
+
+Computes the [hyperbolic sine][hyperbolic-sine] of `x`.
+
+```c
+double out = stdlib_base_sinh( 2.0 );
+// returns ~3.627
+
+out = stdlib_base_sinh( -2.0 );
+// returns ~-3.627
+```
+
+The function accepts the following arguments:
+
+-   **x**: `[in] double` input value.
+
+```c
+double stdlib_base_sinh( const double x );
+```
+
+</section>
+
+<!-- /.usage -->
+
+<!-- C API usage notes. Make sure to keep an empty line after the `section` element and another before the `/section` close. -->
+
+<section class="notes">
+
+</section>
+
+<!-- /.notes -->
+
+<!-- C API usage examples. -->
+
+<section class="examples">
+
+### Examples
+
+```c
+#include "stdlib/math/base/special/sinh.h"
+#include <stdlib.h>
```

Comment:
```suggestion
```

The `stdlib.h` header is not used. For a list of what's in `stdlib.h`, see [here](https://en.wikibooks.org/wiki/C_Programming/stdlib.h). In most cases, we'd use `stdlib.h` when generating random numbers.